### DISCUSSION TITLE
Managing Tones - Saving and Selecting

### DISCUSSION THREAD

**Discussion by wkndcyclr** [soundshed/soundshed-app#65] [Q&A]
1. How do I save a modified tone?  Using a spark mini, with window 10 desktop version of soundshed.  Am able to switch between the 4 tones already saved  on my spark mini, and modify the active one.  How do I save the the updated tone in "My Tones" with a new name

2. How do I select a tone in My Tones - I marked a tone in ToneCloud as a favorite, and see it in My Tones.  How do I select it to use or modify?  I thought the "Play" button in the left column of either MyTones or ToneCloud would do that, but it seems to have no effect - Can't see the settings of that tone in Amp - Just seeing the 4 tones already saved on my spark.

Looking forward to using soundcloud once I get this solved - The controls on the desktop are much easier to adjust than on my phone screen!

**Comment by AxeMaster1** [comment]
New user here, have many of the same questions, I have a recently purchased spark 40 and using either my pc or laptop I am accessing the PG tone cloud tones via soundshed web app. The preview "play" button on the cloud tones works for me but :

I favored some tones on my laptop and then went to my desktop to search and preview more tones and my favs were gone, is there a way to login in somewhere so fav tones are saved and/or on multiple devices?

Also how does one save fav tones from the PG cloud to a preset?

**Comment by webprofusion-chrisc** [comment]
Hi, the web version of the app has now been updated to include a Store Settings button which send the current tone settings to the currently selected channel.

I can confirm that the Play button next to a tone in the tone list should apply it to the amp if the amp is connected. If it just doesn't work  (or if one combination works and others don't) then that's a bug. 

There is currently no sync feature (saving tones on one device and seeing the same list on another device). We did intend to have something like this in the past but the "cloud" options were removed due to the confusion they caused (and their limited functionality at the time).

**Reply by webprofusion-chrisc** [reply]
Just to be clear, this app is not affiliated with Positive Grid and we do not implement any/all of the features the PG app has.

**Reply by wkndcyclr** [reply]
Thanks,  How do I submit as a bug?  What other details do I need to provide

**Reply by webprofusion-chrisc** [reply]
Well you don't need to just test the latest version of https://app.soundshed.com and if it isn't working just list the general steps here. The different tone lists (favourites, tone search etc) may be behaving differently.

**Comment by wkndcyclr** [comment]
That works!  Thanks.
    On Sunday, January 29, 2023 at 09:08:03 PM EST, Christopher Cook ***@***.***> wrote:  
 
 


Well you don't need to just test the latest version of https://app.soundshed.com and if it isn't working just list the general steps here. The different tone lists (favourites, tone search etc) may be behaving differently.
